tcptracer-bpf.c

```diff
@@ -821,6 +841,48 @@ int kretprobe__inet_csk_accept(struct pt_regs *ctx)
 	return 0;
 }
 
+SEC("kprobe/fd_install")
+int kprobe__fd_install(struct pt_regs *ctx)
+{
+	u64 pid = bpf_get_current_pid_tgid();
+	u32 tgid = pid >> 32;
+	unsigned long fd = (unsigned long) PT_REGS_PARM1(ctx);
```

Comment:
I thought LLVM automatically did the parameter mapping (e.g. [like this](https://github.com/weaveworks-plugins/scope-http-statistics/blob/master/ebpf-http-statistics.c#L25)).

Is there a reason for using a macro instead?

Seeing the rest of the kprobes, I realize I am coming late with this remark, sorry.